### ISSUE TITLE
Deploy helm chart to existing ACS cluster

### ISSUE THREAD

**Issue by yfried** [Microsoft/elk-acs-kubernetes#57]
Hi,
I have my own cluster deployed with acs-engine. Is it possible to use this helm chart to deploy ELK to my cluster?

**Comment by yaweiw** [comment]
Yes it works across all K8s clusters. You can refer to: https://github.com/Microsoft/elk-acs-kubernetes/blob/release/helm-charts/start-elk.sh
